test/stored_queries/test_ncbi_tax.py

```diff
@@ -113,10 +127,11 @@ def setUpClass(cls):
         ]
         _create_delta_test_docs('ncbi_taxon', taxon_docs)
         _create_delta_test_docs('ncbi_child_of_taxon', child_docs, edge=True)
-        _create_delta_test_docs('ws_object_version', obj_docs)
-        _create_delta_test_docs('ws_obj_version_has_taxon', obj_to_taxa_docs, edge=True)
-        _create_delta_test_docs('ws_workspace', ws_docs)
-        _create_delta_test_docs('ws_workspace_contains_obj', ws_to_obj, edge=True)
+        create_test_docs('ws_obj_version_has_taxon', obj_to_taxa_docs)
```

Comment:
Are the 2 collections that no longer specify edge=True supposed to be that way?

Reply:
`create_test_docs` does not need that distincton
only `_create_delta_test_docs` uses that param, as it sets fake/default delta fields depending on whether it's an edge or not